include/updater/updater.hpp

```diff
@@ -2,9 +2,20 @@
 #define OSRM_UPDATER_UPDATER_HPP
 
 #include "updater/updater_config.hpp"
+#include "util/log.hpp"
+#include "util/timezones.hpp"
 
 #include "extractor/edge_based_edge.hpp"
+#include "extractor/packed_osm_ids.hpp"
+#include "extractor/query_node.hpp"
+#include "extractor/restriction.hpp"
+#include "util/coordinate.hpp"
+#include "util/packed_vector.hpp"
 
+#include <boost/geometry.hpp>
+#include <boost/geometry/index/rtree.hpp>
```

Comment:
There is no direct dependency on the RTree in the header, it should be fine to remove these and only add includes to `updater.cpp`. All added includes should be removed and moved to `updater.cpp` if it causes compile errors.

This reduces dependencies of files only including `updater.hpp`.